setting ORCHESTRATION_STRATEGY to langchain did not seem to be adhered to

<!--
IF SUFFICIENT INFORMATION IS NOT PROVIDED VIA THE FOLLOWING TEMPLATE THE ISSUE MIGHT BE CLOSED WITHOUT FURTHER CONSIDERATION OR INVESTIGATION
-->
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> We are deploying this repo with gpt-35-turbo model. We therefore assumed ORCHESTRATION_STRATEGY should be set to langchain. We set that in the ARM template but after we deployed we were getting errors in the front end app and noticed in the admin URL config tab that ORCHESTRATION_STRATEGY was set to OPENAI Functions option. 

### Any log messages given by the failure
>2023-10-23T13:55:28.708618922Z: [ERROR]    File "/usr/local/lib/python3.9/site-packages/openai/api_requestor.py", line 700, in _interpret_response
2023-10-23T13:55:28.708646822Z: [ERROR]      self._interpret_response_line(
2023-10-23T13:55:28.708655022Z: [ERROR]    File "/usr/local/lib/python3.9/site-packages/openai/api_requestor.py", line 763, in _interpret_response_line
2023-10-23T13:55:28.708663323Z: [ERROR]      raise self.handle_error_response(
2023-10-23T13:55:28.708671423Z: [ERROR]  openai.error.InvalidRequestError: Unrecognized request argument supplied: functions


### Expected/desired behavior
> manually changing ORCHESTRATION_STRATEGY in config tab and hitting save fixed issue but maybe someone can check the code


### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

--- a/code/backend/batch/utilities/helpers/ConfigHelper.py
+++ b/code/backend/batch/utilities/helpers/ConfigHelper.py
@@ -88,6 +88,7 @@ class ConfigHelper:
 
     @staticmethod
     def get_default_config():
+        env_helper = EnvHelper()
         default_config = {
             "prompts": {
                 "condense_question_prompt": """Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question. If the user asks multiple questions at once, break them up into multiple standalone questions, all in one line.
@@ -204,6 +205,6 @@ Answer: {answer}""",
                 },
             ],
             "logging": {"log_user_interactions": True, "log_tokens": True},
-            "orchestrator": {"strategy": "openai_function"},
+            "orchestrator": {"strategy": env_helper.ORCHESTRATION_STRATEGY},
         }
         return Config(default_config)

--- a/infra/main.bicep
+++ b/infra/main.bicep
@@ -753,3 +753,4 @@ output AZURE_SEARCH_KEY string = useKeyVault ? storekeys.outputs.SEARCH_KEY_NAME
 output AZURE_CONTENT_SAFETY_KEY string = useKeyVault ? storekeys.outputs.CONTENT_SAFETY_KEY_NAME : ''
 output AZURE_SPEECH_SERVICE_REGION string = location
 output AZURE_SPEECH_SERVICE_KEY string = useKeyVault ? storekeys.outputs.SPEECH_KEY_NAME : ''
+output ORCHESTRATION_STRATEGY string = orchestrationStrategy

--- a/infra/main.json
+++ b/infra/main.json
@@ -5,7 +5,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.25.53.49325",
-      "templateHash": "11044000718103660256"
+      "templateHash": "709162983947072789"
     }
   },
   "parameters": {
@@ -6635,6 +6635,10 @@
     "AZURE_SPEECH_SERVICE_KEY": {
       "type": "string",
       "value": "[if(parameters('useKeyVault'), reference(extensionResourceId(format('/subscriptions/{0}/resourceGroups/{1}', subscription().subscriptionId, variables('rgName')), 'Microsoft.Resources/deployments', 'storekeys'), '2022-09-01').outputs.SPEECH_KEY_NAME.value, '')]"
+    },
+    "ORCHESTRATION_STRATEGY": {
+      "type": "string",
+      "value": "[parameters('orchestrationStrategy')]"
     }
   }
 }

